flow/scripts/gui.tcl

```diff
@@ -49,6 +49,7 @@ proc read_timing {input_file} {
   fast_route
 }
 
-if {![env_var_equals GUI_NO_TIMING 1]} {
+if {[env_var_equals GUI_TIMING 1]} {
```

Comment:
Why change the use model?  The docs would also need updating.   I would just mention the opposite "Reading timing which takes a while for large designs.  If you don't need timing use GUI_NO_TIMING =1 for a faster load"

Reply:
Should I post a chatgpt reply?

Reply:
You will have to update the docs then.

Reply:
Will do.

Re ChatGPT blurb; I did try to search for the specific antipattern w.r.t. negative names in variables, but what I can do on the phone is limited.

Reply:
Fixed